modules/ding_nodelist/ding_nodelist.install

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * @file
+ * Installation/uninstallation functions for ding_nodelist module.
+ */
+
+/**
+ * Implements hook_schema().
+ */
+function ding_nodelist_schema() {
+  $schema['ding_nodelist_templates'] = array(
+    'description' => 'Stores nodelist templates.',
+    'fields' => array(
+      'filename' => array(
+        'description' => 'Name of the template file.',
+        'type' => 'varchar',
+        'length' => 255,
+        'not null' => TRUE,
+        'default' => '',
+      ),
+      'title' => array(
+        'description' => 'Template title set by users.',
+        'type' => 'varchar',
+        'length' => 255,
+        'not null' => FALSE,
+        'default' => '',
+      ),
+      'widget' => array(
+        'description' => 'Widget type this template is bind to.',
+        'type' => 'varchar',
+        'length' => 64,
+        'not null' => TRUE,
+        'default' => '',
+      ),
+      'status' => array(
+        'description' => 'Boolean indicating whether the template is enabled by default.',
+        'type' => 'int',
+        'unsigned' => TRUE,
+        'not null' => TRUE,
+        'default' => 0,
+        'size' => 'tiny',
+      ),
+    ),
+    'primary key' => array('filename'),
+    // @todo: define when selects become clear
+    'indexes' => array(),
```

Comment:
If this is configuration is not moved to variables then please update indexes according to queries or remove `@todo`.